pom.xml

```diff
@@ -1018,30 +1018,6 @@ under the License.
 			</build>
 		</profile>
 
-		<profile>
-			<id>java17</id>
-			<activation>
-				<jdk>[17,)</jdk>
-			</activation>
-
-			<build>
-				<pluginManagement>
-					<plugins>
-						<plugin>
-							<groupId>com.diffplug.spotless</groupId>
-							<artifactId>spotless-maven-plugin</artifactId>
-							<configuration>
-								<!-- Current google format does not run on Java 17.
-									 Don't upgrade it in this profile because it formats code differently.
```

Comment:
@zentol I read this line after submitting the patch. Does it mean that google format works differently for Java 8 and Java 17? Or just it cannot work on Java 17 so the result is "different".

Reply:
It...kinda means what it says.

The google java format we use (1.7; _not the spotless version_), does not run on Java 17.
Newer versions that do run format things slightly differently w.r.t. lambdas, but IIRC no longer run on Java 8.